Makefile

```diff
@@ -59,4 +63,9 @@ vagrant-sync-build: build
 vagrant-deploy: vagrant-sync-config vagrant-sync-build
 	export KUBECTL="cluster/kubectl.sh --core" && ./cluster/deploy.sh
 
+.glide.status: glide.yaml
+	if test -f $@; then \
+		glide update --strip-vendor && touch $@; \
```

Comment:
This way glide update only gets triggered if glide.yaml changed. The check for glide install needs to be done every time sync activated. Part of me wants to take this entire section out--and make glide update be something that's only ever done intentionally (and manually).